arch/arm/soc/st_stm32/stm32f1/Kconfig.defconfig.series

```diff
@@ -21,7 +21,7 @@ endif # GPIO_STM32
 
 if I2C && (I2C_1 || I2C_2 || I2C_3)
```

Comment:
Why not if I2C_STM32? The same for the other series.

Reply:
Ah ok, I see.